config/config.go

```diff
@@ -286,12 +286,9 @@ var defaultConf = Config{
 	},
 	LowerCaseTableNames: 2,
 	Log: Log{
-		Level:  "info",
-		Format: "text",
-		File: logutil.FileLogConfig{
-			LogRotate: true,
-			MaxSize:   logutil.DefaultLogMaxSize,
-		},
+		Level:              "info",
```

Comment:
maybe we should also need support

```
curl -X POST -d "log_level=debug" http://0.0.0.0:10080/settings
```
after switch to zap ~?

Reply:
Done.